src/appleseed/renderer/modeling/bsdf/oslbsdf.cpp

```diff
@@ -93,6 +93,11 @@ namespace
             m_glossy_brdf = create_and_register_bsdf(GlossyID, "glossy_brdf");
             m_hair_bsdf = create_and_register_bsdf(HairID, "hair_bsdf");
             m_metal_brdf = create_and_register_bsdf(MetalID, "metal_brdf");
+            m_microfacet_glossy_brdf = create_and_register_bsdf(MicrofacetGlossyID, "microfacet_normal_mapping_glossy_brdf");
+            m_microfacet_blinn_brdf = create_and_register_bsdf(MicrofacetBlinnID, "microfacet_normal_mapping_blinn_brdf");
+            m_microfacet_metal_brdf = create_and_register_bsdf(MicrofacetMetalID, "microfacet_normal_mapping_metal_brdf");
```

Comment:
Somehow, there is no lambertian BRDF anymore here and below (`ll. 420`)

Reply:
This goes for all OSL and Lambertian related comments.
Appleseed is not providing Lambertian OSL (as far as I have seen).
Microfacet normal mapping is a wrapper around existing closures. Therefore, there is no wrapper around Lambertian closure.

NOTE: microfacet normal mapping exists for built-in Lambertian.